packages/stark-ui/src/modules/app-data/components/app-data.component.ts

```diff
@@ -0,0 +1,110 @@
+import { ViewEncapsulation, Component, Inject, Input, OnDestroy, OnInit, Renderer2, ElementRef } from "@angular/core";
+import {
+	StarkLoggingService,
+	STARK_LOGGING_SERVICE,
+} from "@nationalbankbelgium/stark-core";
+import { StarkDOMUtil } from "../../../util/dom/dom.util";
+import { AbstractStarkUiComponent } from "../../../common/classes/abstract-component";
+
+/**
+ * Name of the component
+ */
+const componentName: string = "stark-app-data";
+
+export type StarkAppDataComponentMode = "dropdown" | "menu";
+
+/**
+ * Component to display all the data concerning the current user session including the following info:
+ * - User full name + profile.
+ * - Last access date of the user.
+ * - App version + Environment (PROD, DEV, etc...).
+ */
+@Component({
+	selector: "stark-app-data",
+	templateUrl: "./app-data.component.html",
+	encapsulation: ViewEncapsulation.None,
+	host: {
+		class: componentName
+	}
+})
+export class StarkAppDataComponent extends AbstractStarkUiComponent implements OnInit, OnDestroy {
+	/**
+	 * The mode in which the component should be displayed.
+	 */
+	@Input() public mode?: StarkAppDataComponentMode = "dropdown";
+
+	/**
+	 * If the user image is to be displayed or not.
+	 */
+	@Input() public displayUserImage?: boolean = true;
```

Comment:
You can remove this input now since the image is added by the developer in the transcluded content 😉